src/pperm.c

```diff
@@ -6465,7 +6355,8 @@ Obj FuncOnPosIntSetsPartialPerm(Obj self, Obj set, Obj f)
     }
 
     PLAIN_LIST(set);
-    res = NEW_PLIST_WITH_MUTABILITY(IS_MUTABLE_PLIST(set), T_PLIST_CYC_SSORT, LEN_LIST(set));
+    res = NEW_PLIST_WITH_MUTABILITY(IS_MUTABLE_PLIST(set), T_PLIST_CYC_SSORT,
+                                    LEN_LIST(set));
```

Comment:
Doesn't really matter, but you could use `LEN_PLIST` here, since we just converted `set` to a plist in the line before.

Reply:
Done.